config/locales/diaspora/en.yml

```diff
@@ -449,6 +449,11 @@ en:
       person_multiple_aspects_a: "Yes. Go to your contacts page and click on “My contacts”. For each contact you can use the menu on the right to add them to (or remove them from) as many aspects as you want. Or you can add them to a new aspect (or remove them from an aspect) by clicking the aspect selector button on their profile page. Or you can even just move the pointer over their name where you see it in the stream, and a “hover-card” will appear. You can change the aspects they are in right there."
       delete_aspect_q: "How do I delete an aspect?"
       delete_aspect_a: "In your list of aspects on the left side of the main page, point your mouse at the aspect you want to delete. Click the little “Edit” pencil that appears on the right. Click the “Delete this aspect” button in the box that appears."
+    chat:
+      title: "Chat"
+      contacts_page: "contacts page"
+      add_contact_roster_q: "How do I add contacts to my roster?"
+      add_contact_roster_a: "Go to the %{contacts_page}, select an aspect and click on the chat icon to enable chat for the aspect. %{toggle_privilege} When you create a new aspect you can choose to enable chat for that aspect"
```

Comment:
How about something like:

> ### How do I chat with someone in diaspora*?
> 
> - First you need to enable chat for one of the aspects *[or all of them?]* that person is in.
> - Next, add that person to your 'roster' (the list of people available to chat to)
> - Finally, click their name in the chat interface and start a chat!

Note: I haven't been able to test the chat feature yet, so I've not really an idea about the actual steps. I'm just proposing a more general question with a series of answers for the individual steps required to start chatting with someone; you'll have to write text which is accurate to the actual use case, if what I've said above isn't accurate.

If chat is not automatically activated for each user when a podmin enables it in the pod, the first step might be:

> - If you haven't already enabled the chat feature, go to your user settings page to do this.

Does this sound like a useful approach?